PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -296,23 +301,30 @@ local function CreateCamera()
 	end
 
 	function this:ApplyVRPosition()
+		if not HasVRAPI then
+			return
+		end
 		--we only want this to happen in first person VR
 		local player = game.Players.LocalPlayer
-		if not player or not player.Character
-				or not player.Character:FindFirstChild("HumanoidRootPart")
+		if not player or not player.Character 
+				or not player.Character:FindFirstChild("HumanoidRootPart") 
```

Comment:
We should de morgan's law this:
Factor out the `not` and switch the `or` to `and`